### ISSUE TITLE
Option to pause the video on structured annotations

### ISSUE THREAD

**Issue by JulianKniephoff** [opencast/annotation-tool#454]
This is currently only possible for free text annotations, where you can check a checkbox and have the tool pause the video once you start typing in the appropriate text field. It also automatically resumes when you eventually create the annotation.

**Comment by JulianKniephoff** [comment]
@dagraf I'm not quite sure I understand this correctly. For structured annotations you just want the video paused the moment someone clicks on a label and/or scale value? Should it also auto-resume at some point? When?

**Comment by dagraf** [comment]
@JulianKniephoff Yes, correctly understood. I see no smart solution for an auto-resume. Therefore I would suggest not to implement one. Do you have a suggestion? But please make sure, that hitting the space bar does resume the video.

**Comment by JulianKniephoff** [comment]
No, I don't see a good way to add auto-resume, either. :thinking: 

Just out of curiosity: What exactly is the use case for this feature? That is, what do you expect a user to do once the video is paused after inserting a structured annotation?

**Comment by dagraf** [comment]
Main use case: User has the task to mark a sequence with a structured annotation and to add some thoughts using the comment-function or by writing a freetext annotation at the exact same time.

**Comment by JulianKniephoff** [comment]
OK, thanks, that makes sense. :slightly_smiling_face:

**Comment by dagraf** [comment]
Specification: A click in textbox should NOT stop video. 

Reasoning: 
- if I write a comment, insert it by hitting with enter and click back into the textbox, the video does not stop. But if I insert a comment by clicking on the "Insert" button and then click back into the commentbox, the video does stop. This different behavior can confuse users.
- The description is called differently ("Paus video while writing an freetext annotation").
- The productive tool does work as described above (video does not stop when clicking in the textbox).
